pyproject.toml

```diff
@@ -98,7 +101,6 @@ enable = [
   "bare-except",
   "binary-op-exception",
   "boolean-datetime",
-  "broad-except",
```

Comment:
I would like to keep this enabled and disable it on a line by line basis. The reason is that an overly broad except can hide `TypeError`, `KeyboardInterrupt`, and in some cases even `SyntaxError`. How many places would need to explicitly disable this warning?